app/src/main/java/org/mozilla/fenix/tabstray/TrayPagerAdapter.kt

```diff
@@ -76,30 +70,38 @@ class TrayPagerAdapter(
             }
             PrivateBrowserPageViewHolder.LAYOUT_ID -> {
                 PrivateBrowserPageViewHolder(
-                    itemView,
+                    LayoutInflater.from(parent.context).inflate(viewType, parent, false),
                     tabsTrayStore,
                     browserStore,
                     interactor
                 )
             }
             SyncedTabsPageViewHolder.LAYOUT_ID -> {
                 SyncedTabsPageViewHolder(
-                    itemView,
-                    tabsTrayStore
+                    composeView = ComposeView(parent.context).apply {
+                        layoutParams = ViewGroup.LayoutParams(
+                            ViewGroup.LayoutParams.MATCH_PARENT,
+                            ViewGroup.LayoutParams.MATCH_PARENT
+                        )
```

Comment:
I am wondering if we need this? I haven't seen us needing to do this thus far.

Reply:
It was crashing without this when the Tabs Tray's recyclerview moved to this ViewHolder